WeatherApp/WeatherApp/HomeScreen/WeatherModel.swift

```diff
@@ -0,0 +1,62 @@
+struct WeatherModel: Codable {
```

Comment:
On the topic of code style - refrence Spacing

Every definition (class, protocol, struct, enum...) should have empty line after opening and before closing braces

```swift
struct WeatherModel: Codable {

     let coord: Coord
     let weather: [Weather]
     let base: String
     let main: Main
     let visibility: Int
     let wind: Wind
     let clouds: Clouds
     let dt: Int
     let sys: Sys
     let timezone: Int
     let id: Int
     let name: String
     let cod: Int
     
 }
```

these does not apply to function definitions

Please update it on all of the structs declared here